app/Enums/BackupStatus.php

```diff
@@ -32,6 +32,6 @@ public function getColor(): string
 
     public function getLabel(): string
     {
-        return str($this->value)->headline();
+        return trans('strings.backupstatus.'.str($this->value));
```

Comment:
Spaces should be replaced with underscores! `BackupStatus` = `backup_status` or `Backup Status` for example.

```suggestion
        return trans('strings.backup_status.'.str($this->value));
```